core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2442,7 +2476,8 @@ private void loadMetadataIfNotExist(RpcContext rpcContext, LockedInodePath inode
     if (!inodeExists || loadDirectChildren) {
       try {
         loadMetadataInternal(rpcContext, inodePath, options);
-      } catch (Exception e) {
+      } catch (IOException | InvalidPathException | FileDoesNotExistException | BlockInfoException
```

Comment:
why list every exception?

Reply:
This is because previously it was masking a NPE.. making it difficult to debug.. I figured this way we are more explicit in what we catch.

Reply:
ack